honeybadger/core.py

```diff
@@ -24,7 +24,7 @@ def _send_notice(self, exception, exc_traceback=None, context=None, fingerprint=
         if self.config.is_dev() and not self.config.force_report_data:
             fake_connection.send_notice(self.config, payload)
```

Comment:
I think you should return here as well, so it can be tested.

Reply:
What are your thoughts here? Return a fake uuid generated on client side? Since there's no actual api call made here